azure_sdk/patches/04_armv7m7-imxrt106x.patch

```diff
@@ -0,0 +1,54 @@
+diff -ur '--exclude=*.git' azure-iot-sdk-c-lts_01_2022/CMakeLists.txt azure-iot-sdk-c-lts_01_2022-phoenix/CMakeLists.txt
```

Comment:
shame we need additional patch for one of the targets, hopefully we will be able to make all changes deifne-based in the future (no need to change anything right now)

Reply:
Thanks for the review, I've followed last suggestions (apart from this one), the issue with including libphoenix headers directly has been resolved.